kuma/landing/jinja2/landing/homepage.html

```diff
@@ -112,13 +112,13 @@ <h2 class="offscreen">{{ _('Web Technologies') }}</h2>
 <div class="home-callouts">
   <h2 class="offscreen">{{ _('Open Web and Mobile') }}</h2>
   <div class="column-container center">
-    <div class="column-callout callout-saucelabs">
-      <a href="https://www.saucelabs.com/cross-browser-testing-tutorial?utm_campaign=cbt&amp;utm_medium=banner&amp;utm_source=home-page">
-        <span>
-          <p>{{ _('Test in Firefox and over 800 browser/OS combos with Mozilla and Sauce Labs') }}</p>
```

Comment:
Oh crud :-/ Note taken. I will remember to do this should we ever need to again. Thanks @jwhitlock